doc/nrf/gs_installing.rst

```diff
@@ -644,5 +644,6 @@ Define the required environment variables as follows, depending on your operatin
       See :ref:`zephyr:env_vars_zephyrrc` for more information.
 
 You must also make sure that nrfjprog (part of the `nRF Command Line Tools`_) is installed and its path is added to the environment variables.
+When installing `nRF Command Line Tools`_ on macOS, make sure you download :guilabel:`J-link for x86-version`.
```

Comment:
Same as the comment above, but perhaps skip the explanation if you think it will be too verbose.